textlayout.c

```diff
@@ -531,27 +530,17 @@ int msLayoutTextSymbol(mapObj *map, textSymbolObj *ts, textPathObj *tgret) {
       }
     } else {
       assert(ts->label->maxlength > 0);
-      if(ts->label->wrap != ' '){
-        for(i=0;i<num_glyphs;i++) {
-          /* replace all occurences of the wrap character with a space */
-          if(glyphs.unicodes[i]== ts->label->wrap)
-            glyphs.unicodes[i]= ' ';
-        }
-      }
       if(num_glyphs > ts->label->maxlength) {
+        int num_cur_glyph_on_line = 0; /*count for the number of glyphs on the current line*/
         for(i=0; i<num_glyphs; i++) {
-          if(glyphs.unicodes[i]== ' ')
-            break;
-        }
-        if(i != num_glyphs) { /* we have at least one space on which we can break */
```

Comment:
Loop to check if we have a wrap character is not necessary, as the actual wrapping loop does not do any computation (except counting num_cur_glyph_on_line).